astrophot/image/image_header.py

```diff
@@ -463,42 +301,58 @@ def get_state(self):
         return state
 
     def _save_image_list(self):
+        """
+        Constructs a fits header object which has the necessary information to recreate the Image_Header object.
```

Comment:
```suggestion
        Constructs a FITS header object which has the necessary information to recreate the Image_Header object.
```

In addition to "FITS" being the name, clarifying "FITS" vs "fits" is particularly relevant in a code package about doing fits of models to data in FITS files.

Reply:
haha, very true